app/controllers/suggestion.rb

```diff
@@ -0,0 +1,40 @@
+get '/suggestions' do
+  @suggestions = Suggestion.all
+  erb :"suggestions/index"
+end
+
+get '/suggestions/new' do
+  @suggestion = Suggestion.new
+  erb :"suggestions/index"
```

Comment:
If the new form will be on the index, you can `redirect` here to `"/suggestions"`